bin/runtime/src/lib.rs

```diff
@@ -325,6 +327,21 @@ impl pallet_aleph::Config for Runtime {
     type NextSessionAuthorityProvider = Session;
 }
 
+#[cfg(feature = "liminal")]
+parameter_types! {
```

Comment:
As a matter of sanity check, could you please compare checksum of `aleph-release-runtime` build by this PR and `main` commit ? We expect those to be the same

Reply:
Actually due to common workspace Cargo.lock there are changes expected.